content/shmem_bundle_stop.tex

```diff
@@ -0,0 +1,40 @@
+\apisummary{
+    Stop a communication bundle.
+}
+
+\begin{apidefinition}
+
+\begin{Csynopsis}
+void @\FuncDecl{shmem\_bundle\_stop}@(void);
+void @\FuncDecl{shmem\_ctx\_bundle\_stop}@(shmem_ctx_t ctx);
+\end{Csynopsis}
+
+\begin{apiarguments}
+    \apiargument{IN}{ctx}{A context handle specifying the context on which to
+    perform the optimization. When this argument is not provided, the
+    optimization is performed on the default context.}
+\end{apiarguments}
+
+\apidescription{
+    The \FUNC{shmem\_bundle\_stop} routine provides a hint to the \openshmem
+    library to stop applying bundling-related optimizations.
+}
```

Comment:
We are not specifically making it mandatory for users to call a stop after a start is called. I think, we should make it mandatory to call a stop after a start.

Reply:
And some advice that it's a bad idea  to go into non-shmem code (like a long computation loop) without a stop?